src/main/java/duke/MyList.java

```
@@ -0,0 +1,54 @@
+package duke;
+
+import java.util.*;
+public class MyList {
+    private List<Task> items;
+
+    public MyList() {
+        this.items = new ArrayList<>();
+    }
+
+    public MyList(List<Task> t) {
+        this.items = t;
+    }
+
+    public List<Task> getItemsForSaving() {
+        return this.items;
+    }
+
+    public String addItem(Task t) {
+        this.items.add(t);
+        return "Got it. I've added this task:\n" + t.toString() + "\nNow you have " + this.items.size() + " tasks in the list.";
+    }
+
+    public String getItems() {
+        StringBuilder stringBuilder = new StringBuilder();
+        stringBuilder.append("Here are the tasks in your list:\n");
+        int index = 1;
+
+        for (Task task : this.items) {
+            String itemString = String.format("%d. %s\n", index, task.toString());
+            stringBuilder.append(itemString);
+            index++;
+        }
+
+        return stringBuilder.toString();
+    }
+
+    public String markTask(int index) {
+        if (index < 1 || index > this.items.size()) {
+            return "Number is outside length of list";

```

Would it have been better to throw an exception instead, since this is an error in user input?